host/xdc/integration_failover_test.go

```diff
@@ -134,6 +134,11 @@ func (s *integrationClustersTestSuite) TearDownSuite() {
 	s.cluster2.TearDownCluster()
 }
 
+func (s *integrationClustersTestSuite) decodePayloadsString(ps *commonpb.Payloads) (r string) {
```

Comment:
s.T().Helper() here would be good. And personally I'd call it MustDecodePayloadString but I don't feel all that strongly about it.

Reply:
Good idea about Helper, added. In non-test code I like the `Must` convention but here I think I'd prefer not since the goal is to make tests easier to read.